utils/filesystem/files_test.go

```diff
@@ -1306,6 +1306,46 @@ func TestFilepathStem(t *testing.T) {
 	})
 }
 
+func TestUnzipFileCountLimit(t *testing.T) {
+	fs := NewFs(StandardFS)
+
+	testInDir := "testdata"
+	limits := NewLimits(1<<30, 10<<30, 10)
+
+	t.Run("unzip file above file count limit", func(t *testing.T){
+		testFile := "abovefilecountlimitzip"
+		srcPath := filepath.Join(testInDir, testFile+".zip")
+
+		destPath, err := fs.TempDirInTempDir("unzip-limits-")
+		assert.NoError(t, err)
+		defer func() {
+			if tempErr := fs.Rm(destPath); tempErr != nil {
+					err = tempErr
+			}
+		}()
```

Comment:
You can simplify it this way as you don't do anything with the error
```suggestion
		defer func() {
			_ := fs.Rm(destPath)
		}()
```